### DISCUSSION TITLE
THERMAL RUNAWAY Error

### DISCUSSION THREAD

**Discussion by Darkabstract224** [Jyers/Marlin#1465] [General Discussion]
I've been trying for days now to try to get this firmware to work, it cant make it pass the 4th layer most of the time in any prints. Luckily I have replacements and replaced the heating element and sensor just in case but still I get this error. I looked it up to see if there was a way to solve this and not 10 hours ago someone else posted the exact same issue on reddit. Please fix soon so i dont have to downgrade firmware

**Comment by Darkabstract224** [comment]
4.2.7 w/ BLtouch HS 3x3 jyers 1.3.5b
Reset to default
Store settings
Restore settings
PID@8 cycles at operating temperatures 205 in my case
Leveled bed and adjusted E steps  for my extruder running a micro Swiss
Replaced heating element and temp sensor just in case it was mechanical

made it 2 layers into a benchy before I got thermal runaway again. This time at 195 10 deg cooler than running temp, makes no sense but this is where I’m at.




![3D6BDF5D-E3CD-4DB1-86A0-FB7EB42E91D1](https://user-images.githubusercontent.com/91996944/136212371-55c257a7-d653-43c0-b8ce-75c325ebcc64.jpeg)

**Comment by Darkabstract224** [comment]
I’m looking at configuration.h right now and #Define HEATER_0_MAXTEMP (0-7) are all set to 275, but I’m not printing anywhere near that temperature and #define TEMP_SENSOR_0 has a value of 1. What should I do to these values to fix this issue?

**Reply by Darkabstract224** [reply]
I was having this issue as soon as I installed the firmware for the first time. I was running The previous heating cartridge in thermistor for months. I’ve quadruple checked the connections just in case. I wish it was that simple so I could stop wondering what this is. I made sure everything was OK and I buttoned it back up. I proceeded to do a PID on the hot end for 15 cycles (why not). It fluctuates between 201 and 209 throughout the cycles. Two minutes into a benchy it dips down to 183 and then tries to climb back up and want to get in the mid 190s it fails as a runaway. 2 Heating cartridges and 2 thermistors later it can’t be mechanical Imo but I’ll do anything to get this firmware working correctly.

**Reply by Darkabstract224** [reply]
I’m running a .4mm nozzle too so it shouldn’t dip 25 deg for anything

**Reply by Darkabstract224** [reply]
Steel nozzle. On further inspection I don’t think jyers is at fault I just think it was coincidental timing but I have no idea what my issue is. I saw it fluctuate between 183 and 233 rolling back to marlin 2.0.1. I have no shorts that I can see But what happened is all I want to know. It was doing this before anything was replaced just making that clear

**Reply by Darkabstract224** [reply]
![830F6088-4F47-4288-997D-FD70522CB52C](https://user-images.githubusercontent.com/91996944/136282255-4be608b9-46cd-4851-b95b-cc103cd3361c.jpeg)
![C43CEDE3-E1E6-4E65-A1AB-002E7DEFAF56](https://user-images.githubusercontent.com/91996944/136282262-8b534704-e752-4a8e-902e-ebe58572ce01.jpeg)
![34262D8B-C38C-4421-AA92-A0C9C12CFCE9](https://user-images.githubusercontent.com/91996944/136282265-6fdf8fda-b281-4d6a-a1a6-6017a72651db.jpeg)

![55128BEC-F801-4995-9B99-A80357B1EF60](https://user-images.githubusercontent.com/91996944/136282263-c5edd7c2-7a22-4ac2-a67e-f57052b55654.jpeg)

**Reply by Darkabstract224** [reply]
All right I will check it out. I’m about to replace the thermistor and heating cartridge again

**Reply by Darkabstract224** [reply]
Solved: I guess my heating cartridge and thermistor failed as soon as I updated to jyers, replaced it with faulty equipment heating/thermistor or both. I naturally assumed it was the firmware I just updated to but it was extreme coincidence I suppose because it’s been a consistent 204 to 205 C for 10 minutes now after replacing for the second time. Now I’m going to try jyers once again with normal running equipment lol.
What a headache

**Comment by Redleg1096** [comment]
Well, in case no one mentioned it,
PUT ON A THERMAL SOCK!!!!!
Also, use thermal paste on the heater cartridge.
I used E3D's Boron Nitrite Paste.  Works great.
